config/prow/plugins.yaml

```diff
@@ -1871,3 +1875,12 @@ external_plugins:
     - issue_comment
     - pull_request
     endpoint: http://cherrypicker
+  containerd/containerd:
+  - name: needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+  - name: cherrypicker
```

Comment:
Does the cherrypicker work based on the current labels that is being used in containerd?

Reply:
I'm not sure, and I'm not sure if it'll really work much given the big code organization changes that have happened in main compared to 1.x branches.  But I figured it was worth enabling to try.  (We can always change the labels in containerd too.)